tests/test_qpex.py

```diff
@@ -105,7 +105,13 @@ def create_qp(self):
         self.qp = create_qp_ex(self, e.IBV_QPT_RC, e.IBV_QP_EX_WITH_RDMA_WRITE)
 
     def create_mr(self):
-        self.mr = u.create_custom_mr(self, e.IBV_ACCESS_REMOTE_WRITE)
+        try:
+            access_flags = e.IBV_ACCESS_REMOTE_WRITE
+            self.mr = u.create_custom_mr(self, access_flags)
```

Comment:
hm.. why not to include this try-catch check inside the create_custom_mr itself?
This will cover all current/future similar scenarios that call create_custom_mr.

Reply:
Done.